ocrd_anybaseocr/cli/ocrd_anybaseocr_layout_analysis.py

```diff
@@ -165,6 +215,7 @@ def process(self):
                 """ % model_path)
             sys.exit(1)
```

Comment:
so far we were not aware that GPU should not be a hard requirement. in previous meetings we specifically asked which hardware we can assume to be available

Reply:
ok, maybe I got this wrong. Better ask officially. But from a user perspective, I think you should always allow running on CPU only as well (even if this will take 10 times as long), especially if it's possible by default on your framework.